src/solvers/dgsem_tree/subcell_limiters_2d.jl

```diff
@@ -52,6 +56,172 @@ function (limiter::SubcellLimiterIDP)(u::AbstractArray{<:Any, 4}, semi, dg::DGSE
     return nothing
 end
 
+@inline function calc_bounds_2sided!(var_min, var_max, variable, u, t, semi)
```

Comment:
```suggestion
@inline function calc_bounds_twosided!(var_min, var_max, variable, u, t, semi)
```
We try to avoid having digits as abbreviations unless there is a strong motivation (maybe there is?)

Reply:
No. There is no reason. Thanks.

Reply:
Done in [5adf14c](https://github.com/trixi-framework/Trixi.jl/pull/1648/commits/5adf14c9d8bbb5cd043e36ad8067dfcd22880cb4)